include/pmacc/traits/HasIdentifiers.hpp

```diff
@@ -46,10 +46,13 @@ namespace pmacc
         template<typename T_Object, typename T_SeqKeys>
         struct HasIdentifiers
         {
-            using SeqHasIdentifiers = typename bmpl::transform<T_SeqKeys, HasIdentifier<T_Object, bmpl::_1>>::type;
+            using SeqHasIdentifiers =
+                typename boost::mpl::transform<T_SeqKeys, HasIdentifier<T_Object, boost::mpl::_1>>::type;
```

Comment:
offline answered question: `bmpl` -> `boost::mpl` avoids short notation which is defined somewhere.